code-highlighter.html

```diff
@@ -225,18 +225,21 @@
       _splitHunk: function (e) {
         var path = this._getComposedPath(e)[0];
         var lineNumber = Number(path.getAttribute('index')) + 1;
-        if (path.localName === 'iron-icon' && lineNumber !== this._amountLines) {
-          var newElem = {};
-          var x = this.code.split('\n');
-          newElem.code = x.slice(lineNumber, this.code.length - 1).join('\n');
-          newElem.beforeStart = path.getAttribute('before');
-          newElem.afterStart = path.getAttribute('after');
-          newElem.index = this.index;
-          this.code = x.slice(0, lineNumber).join('\n');
-          this.fire('add-hunk', newElem);
+        if (path.localName === 'iron-icon') {
+          if (lineNumber !== this._amountLines) {
+            var newElem = {};
+            var x = this.code.split('\n');
+            newElem.code = x.slice(lineNumber, this.code.length - 1).join('\n');
+            newElem.beforeStart = path.getAttribute('before');
+            newElem.afterStart = path.getAttribute('after');
+            newElem.index = this.index;
+            this.code = x.slice(0, lineNumber).join('\n');
+            this.fire('add-hunk', newElem);
```

Comment:
Or `click-line`. This is more general and leaves handling of the 'last line clicked' case to the parent element. So then `code-with-comment` would decide that a comment has to be added when the last line is clicked.